src/test/java/rx/internal/operators/OperatorIgnoreElementsTest.java

```diff
@@ -92,7 +92,7 @@ public void call(Integer t) {
                     }
                 })
                 //
-                .ignoreElements()
+                .<Integer>ignoreElements()
```

Comment:
Would be nice to check binary compatibility as part of CI, there are some checker tools that we can include into the build process. Probably, it deserves a separate issue.